lib/validation_msg_deso_txn_test.go

```diff
@@ -0,0 +1 @@
+package lib
```

Comment:
What's going on in this file? If you're setting up for some future use, please leave a comment in here.

Reply:
Oh, this PR was a draft. We've got tests here now.